### PR TITLE
55 nrf connect

### PR DESCRIPTION
**Pull request by EliHaugu**
requirements.txt
* Added yaml

data_manager
* Added migration to add default nodes for nrf connect and custom script

device_connection
* Add flow connection endpoint, please check that the error handling works here (the FlowDeviceConnectionView)
* Rest is just formatting and also some missing JsonResponses, please check through that no View in the views.py returns nothing. That is, SerialDeviceConnectionView, AndroidDeviceConnectionView, APIConnectionView and the aforementioned FlowDeviceConnectionView
* Added nRFConnectionView, currently not in use but kept
* Note that FlowDeviceConnection does not check bluetooth mac address stuff anymore since we decided to put that into a default node

server_comm
* Add yaml file with function format for default nodes
* Otherwise just formatting, again the complaints on " and '

test_runner
* apps.py and urls.py is just formatting
* nrf_scripts folder contains xml scripts for running nrf scripts, and can add custom scripts
* nrf_connect.py runs the different scripts by using a shell script
* test.sh is the shell script
* In views.py added nrf connection function
* In views.py added run command
* Added default_nodes.py script for running default nodes

### CODE REVIEW

**Comment by EliHaugu** [comment]
Don't know why it says I added #93 ?

**Comment by EliHaugu** [comment]
Don't know why the flake8 check fails, works on my end?

File: server/server_comm/test_runner/views.py [hunk 1]
```diff
@@ -91,16 +92,61 @@ def check_device_connections(self, flow_id):
...
+
+        nrf_devices = self.get_nrf_devices(devices)
+
+        for device in nrf_devices:
+            dev_id = device.device_id
+            mac = device.communication_ids.get("mac_address")
+            res = run_check_connection(dev_id, mac)
+
+            if res.get("status") == "error":
+                raise RuntimeError(
```
Comment (EliHaugu) [inline server/server_comm/test_runner/views.py:134]: where is the missing ")"????

**Comment by EliHaugu** [comment]
Force pushed rebase

**Comment by EliHaugu** [comment]
FUCK MENTE IKKE ADDE DEN BITEN ENDA skulle splitte det opp i fine commits shit

**Review by haakon98 — APPROVED** [review]

**Pull request merged**